### PR TITLE
fix: gallery on product page

### PR DESCRIPTION
**Pull request by dead-playmod**
## Description
Fixes issue with image gallery on product page which didn't work unless you refresh while being on product page.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

#### Changelog
- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.

#### Tests
- [ ] I have written test cases for my code
- [ ] I have tested my Pull Request on production build and (to my knowledge) it works without any issues
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

> I tested manually my code, and it works well with both:
- [ ] Default Theme
- [ ] Capybara Theme

#### Code standards
- [ ] My code follows the code style of this project.

- [ ] I have followed [naming conventions](https://github.com/kettanaito/naming-cheatsheet)

#### Docs
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/theme/pages/Product.vue [hunk 1]
```diff
@@ -179,9 +179,7 @@ export default {
     const { addItem, loading } = useCart();
     const { reviews: productReviews, search: searchReviews, addReview } = useReview('productReviews');
 
-    const product = computed(() => {
-      return products.value.products && productGetters.getFiltered(products.value.products, { master: true, attributes: context.root.$route.query })[0];
-    });
```
Comment (senghe) [inline packages/theme/pages/Product.vue:184]: Is this change needed?

Comment (dead-playmod) [reply packages/theme/pages/Product.vue:184]: > Is this change needed?

It isn't but in other cases computed was used without return and it isn't required, so it's mostly to make it similar to other computed properties.

**Review by senghe — APPROVED** [review]
